docs/tutorial/image_keypoint_detection.md

```diff
@@ -0,0 +1,226 @@
+Export `DATA_DIR` to make sure dataset loader can locate it properly.
+
+    export DATA_DIR=/storage/dataset
+
+Optionally, you can also export `OUTPUT_DIR`. if not, default 
+`OUTPUT_DIR` will be `saved/` under blueoil root path.
+
+    export OUTPUT_DIR=/storage/saved
+
+In this tutorial, we assume we have exported both variables as above.
+
+## Generate a configuration file
+
+Note: Below instructions assume your current path is blueoil root path.
+
+Generate your model configuration file interactively by running the `python blueoil/cmd/main.py init` command.
+
+    $ PYTHONPATH=.:lmnet:dlk/python/dlk python blueoil/cmd/main.py init
+
+Below is an example of initialization.
+
+```
+#### Generate config ####
+your model name ():  keypoint_detection_demo
+choose task type  keypoint_detection
+choose network  LmSinglePoseV1Quantize
+choose dataset format  Mscoco for Single-Person Pose Estimation
+training dataset path:  MSCOCO_2017
+set validation dataset? (if answer no, the dataset will be separated for training and validation by 9:1 ratio.)  yes
+validation dataset path:  MSCOCO_2017
+batch size (integer):  4
+image size (integer x integer):  160x160
+how many epochs do you run training (integer):  100
+select optimizer:  Adam
+initial learning rate:  0.001
+choose learning rate schedule ({epochs} is the number of training epochs you entered before):  '3-step-decay-with-warmup' -> warmup learning rate 1/1000 in first epoch, then train the same way as '3-step-d
+enable data augmentation?  Yes
+Please choose augmentors:  done (6 selections)
+apply quantization at the first layer?  no
+```
+
+If configuration finishes, the configuration file is generated in the `{Model name}.yml` under current directory.
+It should be `keypoint_detection_demo.yml` if you use above example initialization.
+
+
+When you wnat to create config yaml in specific filename or directory, you can use `-o` option.
+
+    $ PYTHONPATH=.:lmnet:dlk/python/dlk python blueoil/cmd/main.py init -o ./configs/my_config.yml
+
+## Train a network model
+
+Train your model by running `python blueoil/cmd/main.py train` with a model configuration.
+Feel free to replace `keypoint_detection_demo.yml` by your custom config file.
+
+    $ PYTHONPATH=.:lmnet:dlk/python/dlk python blueoil/cmd/main.py train -c keypoint_detection_demo.yml
+    
+    
+
+When training has started, the training log and checkpoints are generated under `${OUTPUT_DIR}/{Mode name}_{TIMESTAMP}`,
+i.e. `/storage/saved/keypoint_detection_demo_{TIMESTAMP}` here.
+
+Training runs on the TensorFlow backend. So you can use TensorBoard to visualize your training process.
+
+    $ tensorboard --logdir=${OUTPUT_DIR}/keypoint_detection_demo_{TIMESTAMP} --port {Port}
+
+- input / labels_heatmap / output_heatmap
+<img src="../_static/keypoint_detection_heatmap.png">
+
+- visualize_labels, visualize_output
+<img src="../_static/keypoint_detection_visualize.png">
+
```

Comment:
It's better to add some metrics screenshots.

Reply:
Added.